lib/Minz/Error.php

```diff
@@ -4,30 +4,32 @@
 /**
  * MINZ - Copyright 2011 Marien Fressinaud
  * Sous licence AGPL3 <http://www.gnu.org/licenses/>
-*/
+ */
 
 /**
  * The Minz_Error class logs and raises framework errors
  */
-class Minz_Error {
+class Minz_Error
+{
 	public function __construct() {}
 
 	/**
-	* Permet de lancer une erreur
-	* @param int $code le type de l'erreur, par défaut 404 (page not found)
-	* @param string|array<'error'|'warning'|'notice',list<string>> $logs logs d'erreurs découpés de la forme
-	*      > $logs['error']
-	*      > $logs['warning']
-	*      > $logs['notice']
-	* @param bool $redirect indique s'il faut forcer la redirection (les logs ne seront pas transmis)
-	*/
-	public static function error(int $code = 404, string|array $logs = [], bool $redirect = true): void {
+	 * Launches an error
+	 * @param FreshRSS_HttpResponseCode $code error type
+	 * @param string|array<'error'|'warning'|'notice',array<string>> $logs error logs broken down by form
+	 *      > $logs['error']
+	 *      > $logs['warning']
+	 *      > $logs['notice']
+	 * @param bool $redirect indicates whether to force redirection (logs will not be transmitted)
+	 * @throws Minz_ConfigurationException
+	 */
+	public static function error(FreshRSS_HttpResponseCode $code, string|array $logs = [], bool $redirect = true): void {
```

Comment:
This looks like a breaking change. Is that intentional? Based on the old code I'd expect to see something like:

```suggestion
	public static function error(FreshRSS_HttpResponseCode $code = FreshRSS_HttpResponseCode::HTTP_404_NOT_FOUND, string|array $logs = [], bool $redirect = true): void {
```